config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  root "trips#index"
+
+  resources :trips, :drivers, :passengers
```

Comment:
Do you need a route for `get '/trips'`?  What about `/trips/new`?  Will you ever bring up a trip form without a passenger?